### ISSUE TITLE
Try to break lines more accurately.

### ISSUE THREAD

**Issue by LuisMayo** [LuisMayo/objection_engine#55]
In English lowercase, sometimes up to half of the box is empty. Space should be better used when possible

**Comment by lens0021** [comment]
I can reproduce it by printing a long word without whitespace.

https://user-images.githubusercontent.com/28209361/153135914-f7b8b714-85a4-474f-9e61-c0cdd12493b1.mp4

**Comment by LuisMayo** [comment]
Yup, it also gives trouble with certain languages were spaces are not used so commonly. I'm going to open anew issue for the space issue
